src/mpi4py/MPI.pyi

```diff
@@ -538,11 +555,14 @@ class Info:
     def __contains__(self, value: str) -> bool: ...
     @classmethod
     def Create(cls) -> Info: ...
+    @classmethod
+    def Create_env(cls, args: Optional[Sequence[str]] = None) -> Info: ...
     def Free(self) -> None: ...
     def Dup(self) -> Info: ...
-    def Get(self, key: str, maxlen: int = -1) -> Optional[str]: ...
```

Comment:
I don't know if it matters at all, but you could use a tighter bound here:
MPI_MAX_INFO_KEY Maximum length of an info key
MPI_MAX_INFO_VAL Maximum length of an info value

Reply:
Please note you are commenting on a line I'm removed. 

The new signature is
```python
def Get(self, key: str) -> Optional[str]: ...
```
Unfortunately, I believe Python's typing does not allow you to specify size bounds for string.

Also note that the `Info.Get()` method is still there for backward compatibility. I added a new method `Info.Get_string()`, which is implemented with the new MPI-4 routine `MPI_Info_get_string()`.